datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -2906,6 +2932,26 @@ input DashboardUpdateInput {
     editableProperties: DashboardEditablePropertiesUpdate
 }
 
+"""
+Arguments provided to update a Notebook Entity
+"""
+input NotebookUpdateInput {
+    """
+    Update to ownership
+    """
+    ownership: OwnershipUpdate
+
+    """
+    Update to tags
+    """
+    tags: GlobalTagsUpdate
```

Comment:
I believe we added terms as well. so will need update for terms

Reply:
Do you mean glossaryTerms update? Looks like there is no data entity that allows updating glossaryTerms?

Reply:
Sorry about the confusion. You are right, there is no update for terms. Resolving comment!